app/views/layouts/application.html.erb

```diff
@@ -10,7 +10,9 @@
     <%#= stylesheet_pack_tag 'application', media: 'all' %> <!-- Uncomment if you import CSS in app/javascript/packs/application.js -->
   </head>
   <body>
-    <%= render 'shared/navbar' %>
+    <% if controller_name != 'pages' && action_name != 'home' %>
```

Comment:
Why did you change this?

Reply:
This will make sure that the navbar will be shown on every page except the landingpage